File: pet-service/src/main/java/com/phorus/petservice/model/dbentities/Pet.java
package com.phorus.petservice.model.dbentities;

import lombok.*;
import lombok.experimental.SuperBuilder;
import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Table;

@Table("pets")
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@SuperBuilder
@EqualsAndHashCode
public class Pet {

    @Id
    private Long id;

    private String name;
    private Integer age;
    private Long userId; // Owner
}

File: pet-service/src/main/kotlin/com/phorus/petservice/model/dbentities/Pet.kt
package com.phorus.petservice.model.dbentities

import org.springframework.data.annotation.Id
import org.springframework.data.relational.core.mapping.Table


@Table("pets")
class Pet(
    @Id
    var id: Long? = null,
    var name: String? = null,
    var age: Int? = null,
    var userId: Long? = null, // Owner
)